src/main/scala/mesosphere/marathon/Main.scala

```diff
@@ -52,6 +54,14 @@ class MarathonApp extends App {
       new HttpModule(conf) {
         // burst browser cache for assets
         protected override val resourceCacheControlHeader = Some("max-age=0, must-revalidate")
+
+        override def provideHandlerCollection(instrumentedHandler: InstrumentedHandler,
```

Comment:
The problem is the order of the handler collection. The overridden method has the same handlers in different order. The instrumentedHandler needs to go before the resourceHandler (because it will try to load a resource directly, before we can handle the request in the controller)